test/Microsoft.Framework.Runtime.Tests/VersionUtilityFacts.cs

```diff
@@ -118,18 +119,20 @@ public void ShortFrameworkNamesAreCorrect(string longName, string version, strin
 
         [Theory]
         [InlineData("dnx46", "dotnet,dnx46", "dnx46")]
+
+        // Portable frameworks should use the old matching system based on scoring to ensure the best PCL is chosen
+        [InlineData(".NETPortable,Version=v4.0,Profile=Profile6", "portable-net40+sl4+win8,portable-net40+win8", "portable-net40+win8")]
+
+        // Profile restrictions should be honored (yes, I know net46 client doesn't exist, just for testing :))
+        [InlineData("net46-client", "net46,net45-client,net40", "net45-client")]
```

Comment:
Before this change, `net46` would be taken. Granted, this is a totally arbitrary contrived case, but a project with a framework constrained by a profile cannot generally take profile-less content.